concrete_datastore/api/v1_1/urls.py

```diff
@@ -113,6 +114,11 @@
         GenerateSecureTokenApiView.as_view(),
         name='generate-secure-token',
     ),
+    re_path(
+        r'process/register/',
+        ProcessRegisterApiView.as_view(),
+        name='process-register',
```

Comment:
the name of the view is not explicit enough. 

The view above is named "generate ... token" as it is used to generate a token. This view does not "process" anything. Maybe "register-as-process"

Reply:
name changed to `register-as-process `